server/index.js

```diff
@@ -163,10 +163,12 @@ app.get('/drink/:drinkId', (req, res) => {
         })
         .then(singleDrinkIngredients => {
           let drinkIngredients = {};
+          // let imageUrl = 'http://assets.absolutdrinks.com/drinks/300x400/${strID}.png'
 
           singleDrinkIngredients.forEach(drinkIngredient => {
-            let ingredientName = drinkIngredient.Ingredient.dataValues.name;
+            let ingredientName = drinkIngredient.ingredient.dataValues.name;
             let ingredientMeasure = drinkIngredient.dataValues.measure;
+            console.log('****Drink IDs', drinkIngredient.ingredient.dataValues.id)
```

Comment:
This one is just here for testing purposes. That way we know what IDs go with a drink and we can test out the POST req above.